lib/services/http_service/http_service.dart

```diff
@@ -118,7 +118,7 @@ class HttpService {
     final String proposalsURL =
         'https://api.telos.eosindex.io/v1/chain/get_table_rows';
 
-    final String minimumStake = "1.0000 SEEDS";
+    // final String minimumStake = "1.0000 SEEDS";
 
```

Comment:
why?

Reply:
Only proposals with staked enough tokens are considered to be active, but since function changed to access other types of proposals, this should not be filtered by active proposals at the moment